client/components/orderCard.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+import {AddToCart} from './index'
+
+const OrderCard = props => {
+  return (
+    <li>
+      <Link to={`./orders/${props.order.id}`}>
```

Comment:
Can you update the Link components here? The `to={/path/to/component}` here needs to start with just the `/`, not the `./` -- the `./` is for reference file directories and `/` denotes url paths.